tardis/energy_input/gamma_ray_channel.py

```diff
@@ -167,3 +167,80 @@ def create_isotope_decay_df(cumulative_decay_df, gamma_ray_lines):
     )
 
     return isotope_decay_df
+
+
+def distribute_packets(isotope_decay_df):
```

Comment:
pandas dataframe.sample function do the normalized weights sampling, so maybe this function is not needed to save computational resource.

Reply:
Resolved!